staging/src/k8s.io/apiserver/pkg/storage/etcd3/watcher_test.go

```diff
@@ -224,6 +224,12 @@ func TestWatchFromNoneZero(t *testing.T) {
 }
 
 func TestWatchError(t *testing.T) {
+	// we need to check for watch errors so do not panic the server
```

Comment:
> @deads2k - do you think we should care about this race?

I think we can restructure the test to catch the expected behavior.  If we're set to panic, use a defer/recover construct.  If we're set to not-panic, expect the error.